integrations/Podsights/constants.js

```diff
@@ -0,0 +1,160 @@
+const NAME = "PODSIGHTS";
+const CNameMapping = {
+  [NAME]: NAME,
+  Podsights: NAME,
+  PodSights: NAME,
+  "pod Sights": NAME,
+  "Pod Sights": NAME,
+  "pod sights": NAME,
+  "POD SIGHTS": NAME,
+  "Pod sights": NAME,
+};
+
+const LEAD_EVENT = [
+  {
+    destKey: "value",
+    sourceKeys: "value",
+  },
+  {
+    destKey: "currency",
+    sourceKeys: "currency",
+  },
+  {
+    destKey: "type",
+    sourceKeys: "type",
+  },
+  {
+    destKey: "category",
+    sourceKeys: "category",
+  },
+];
+
+const PRODUCT_EVENT = [
```

Comment:
I hope we have mentioned in docs that for product events, we are expecting product properties to be present in message.properties?

Reply:
Should we consider this case:
```
product: [
 {
     product_name: "",
     product_type: "",
     product_vendor: "", 
     product_id: "",
 },
 {
     product_name: "",
     product_type: "",
     product_vendor: "", 
     product_id: "",
 } 
]
```
And from this, should we'll fetch each item detail and call the product_event same as the product object size?
@shrouti1507

Reply:
from each product object, we should fetch only these keys and we have to mention the mapping in the doc.

Reply:
```product: [
 {
     product_name: "",
     product_type: "",
     product_vendor: "", 
     product_id: "",
 },
 {
     product_name: "",
     product_type: "",
     product_vendor: "", 
     product_id: "",
 } 
]
```
this case need not to be considered. If this case arrives, we will omit the products array

Reply:
done